src/game/SpellEffects.cpp

```diff
@@ -5457,6 +5457,33 @@ void Spell::EffectScriptEffect(SpellEffectIndex eff_idx)
                     unitTarget->RemoveAurasAtMechanicImmunity(IMMUNE_TO_ROOT_AND_SNARE_MASK, 30918, true);
                     break;
                 }
+                
+                
+                case 32027:				// Expedition Flare
+			{
+				if (!unitTarget)
+					return;
+
+				uint32 creature_id;
+				uint32 rand = urand(0, 99);
+
+				//summon Expedition Preserver or Expedition Scout 
+				if (rand < 50)
+					creature_id = 18194;
+				else 
+					creature_id = 18126;
+		
+				float angle = unitTarget->GetOrientation();
+			
+				Creature* pCreature = m_caster->SummonCreature(creature_id, 0.0f, 0.0f, 0.0f, angle, TEMPSUMMON_TIMED_OOC_OR_DEAD_DESPAWN, 60000, true);
```

Comment:
There is a ton of serverside spells that summon creatures (i have a suspicion blizz summons all like that), problem is, a shitton of these serverside spells are not in client spell.dbc, hence we have to use summonCreature function, i.e. the usage is correct here